src/index.css

```diff
@@ -0,0 +1,593 @@
+@import url("https://fonts.googleapis.com/css2?family=Cormorant:wght@300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+body {
+  min-width: 300px;
+  margin: 0;
+  padding: 0;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  font-family: "Cormorant", serif;
+}
+
+h2 {
+  font-size: 30px;
+}
+
+p {
+  font-family: "Roboto", sans-serif;
+  font-size: 14px;
+}
+
+/* header section and nav bar start here  */
+.navbar{
+  background-color: var(--colour-sage);
+  color: var(--colour-white);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  padding-left: 20px;
+}
+
+
+
+/* .nav-elements {
+  display: none;
+  display: flex;
+  flex-flow: column nowrap;
+  list-style-type: none;
+  background-color: var(--colour-sage);
+  position: fixed;
+  top: 0;
+  right: 0;
+  height: 50vh;
+  width: 200px;
+  padding-top: 3.5rem; 
+} */
+
+ul {
+  display: flex;
+  /* flex-flow: nowrap; */
+  list-style-type: none; 
+}
+
+li {
+  padding: 10px;
+}
+
+.nav-link {
+  color: var(--colour-white);
+  text-decoration: none;
+}
+
+li:hover {
+  background-color: var(--colour-biscuit);
+}
+
+/* ----------hamburger-------  */
+
+
+
+/* footer section starts here */
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--colour-sage);
+  color: var(--colour-white);
+  margin-top: 50px;
+  padding: 20px;
+}
+
+.footer-logo {
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+}
+
+.footer-text {
+  min-width: 200px;
+  padding-right: 20px;
+}
+
+.social-logo {
+  display: flex;
+  justify-content: flex-end;
+  padding: 20px;
+}
+
+/* ---------------------------  */
+/* Home page starts here  */
+.home-hero {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./images/Home/Hero.png");
+  font-family: "Cormorant", serif;
+  min-height: 500px;
+  background-size: cover;
+}
+
+.hero-title {
+  color: var(--colour-white);
+  padding: 0px 50px;
+  font-size: 88px;
+}
+
+/* about section starts here  */
+.blend-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+}
+
+.item {
+  flex-basis: 100%;
+  /* border: 1px solid black; */
+  padding: 20px;
+}
+
+.blend-img {
+  max-width: 100%;
+  height: auto;
+  text-align: center;
+}
+
+.line{
+  background-color: var(--colour-biscuit);
+  width: 30%;
+  height: 3px;
+  margin-left: 20px;
+}
+
+.hand-picked-txt {
+  color: var(--colour-biscuit);
+  font-family: "Cormorant", serif;
+  font-size: 61px;
+  margin: 20px;
+}
+
+/* wise-leaf section starts here*/
+.teapot-img,
+.flowers-img {
+  width: 100%;
+  height: auto;
```

Comment:
This needs to be `height: 100%` for the image to fill the whole space allocated. 

The parent of this image, `wiseleaf-img1`, is correctly set to the right section of the grid, but the image is not filling that space (it's not filling its parent). 

Set the height to 100% and also remove the `margin-bottom` on the parent - it also interferes with how the image fills the space.